endpoints/cookie_sync_test.go

```diff
@@ -647,18 +657,109 @@ func TestCookieSyncParseRequest(t *testing.T) {
 			givenCCPAEnabled: true,
 			expectedError:    "Failed to read request body",
 		},
+		{
+			description: "Request where account level defaults are applied: - MaxLimit and Default Coop",
+			givenBody: strings.NewReader(`{` +
+				`"bidders":["a", "b"],` +
+				`"gdpr":1,` +
+				`"gdpr_consent":"anyGDPRConsent",` +
+				`"us_privacy":"1NYN",` +
+				`"limit":42,` +
+				`"account":"TestAccount"` +
+				`}`),
```

Comment:
I recommend removing gdpr, gdpr_consent, and us_privacy from the request. It's ideal for the test setup to be as simple as possible and these values don't seem to have an impact on the area of code you are testing. Look to a test case like "Cooperative True - Default False" as an example.